src/core/server/ui_settings/ui_settings_client.ts

```diff
@@ -116,7 +71,7 @@ export class UiSettingsClient implements IUiSettingsClient {
     this.log = log;
   }
 
-  getDefaults() {
+  getRegistered() {
     return this.defaults;
```

Comment:
Should we rename this private value? If so, I think we should rename it in `UiSettingsServiceOptions` as well. Also it appears to expose a mutable value.

Reply:
> Should we rename this private value

We should. Surprisingly that's quite a lot of work. I have the renaming in following steps https://github.com/elastic/kibana/issues/44994

> Also it appears to expose a mutable value.

Each client get new version of `defaults`.https://github.com/elastic/kibana/pull/48413/files/d5a0b8b7832393b9778337b5c2c3f1b8bde59926#diff-39a763c69e856858c363b176d7490f48R64
I thought to make it immutable but decided not to introduce additional performance penalty and get by with `Readonly` types. https://github.com/elastic/kibana/pull/48413/files/d5a0b8b7832393b9778337b5c2c3f1b8bde59926#diff-51f25153bbdc2a48df987a3738119efbR12